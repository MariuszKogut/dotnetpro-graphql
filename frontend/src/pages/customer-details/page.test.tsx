import React from 'react'
import { render } from '@testing-library/react'
import CustomerDetailsPage from './page'

jest.mock('react-router', () => ({
  useParams: jest.fn().mockImplementationOnce(() => ({
    id: '4711',
  })),
}))

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
    history: jest.fn(),
  }),
}))

describe('CustomerDetailsPage', () => {
  it('renders without errors', () => {
    const { container } = render(<CustomerDetailsPage />)
    expect(container).toMatchSnapshot()
  })
})
