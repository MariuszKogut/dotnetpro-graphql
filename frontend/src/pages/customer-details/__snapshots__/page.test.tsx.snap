// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerDetailsPage renders without errors 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="py-3 row"
    >
      <div
        class="col"
      >
        <div
          class="fade alert alert-info show"
          role="alert"
        >
          <div
            class="spinner-border"
            role="status"
          >
            <span
              class="sr-only"
            >
              Kunde wird geladen...
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
